src/screens/home/home.tsx

```diff
@@ -1,21 +1,39 @@
 import React from 'react';
-import {View, StyleSheet} from 'react-native';
+import {View, StyleSheet, ListRenderItemInfo} from 'react-native';
 import {CustomButton} from '../../components/custom-button';
+import {UserItem} from '../../components/user-item';
+import {userList, UserList} from '../../data/user-list';
 import {useAuth} from '../../contexts/auth';
+import {FlatList} from 'react-native-gesture-handler';
+import {SeparatorItem} from '../../components/separator-item';
 
 export const Home = () => {
   const {signOut} = useAuth();
 
+  const renderItem = ({item}: ListRenderItemInfo<UserList>) => {
+    return <UserItem {...item} />;
+  };
+
   return (
     <View style={styles.root}>
-      <CustomButton text="Voltar" onPress={signOut} />
+      <FlatList
+        ItemSeparatorComponent={SeparatorItem}
+        keyExtractor={item => item.email}
+        data={userList}
+        renderItem={renderItem}
+      />
+      <CustomButton text="Sair" onPress={signOut} />
```

Comment:
Esse botão não vai acabar aparecendo quando o usuário está deslogado tbm?

Reply:
Não porque ele so existe na tela home, na signin (que é quando o usuario esta deslogado) ele não existe

Reply:
Mas a home não é a listagem de usuários? Ela não tem que aparecer quando o usuário está logado?

Reply:
sim, mostra quando o usuario esta logado, e se ele clicar em sair, ele é redirecionado pra tela de sign in onde esse botao nao existe